pkg/resource/aws/testdata/acc/aws_security_group/terraform.tf

```diff
@@ -72,26 +67,6 @@ resource "aws_security_group" "sg-bis-tutu" {
     }
 }
 
-resource "aws_security_group_rule" "tutu-egress" {
```

Comment:
I don't get why do we have diff on theses acceptance test here

Reply:
I remove those security group rules since they are useless in this acceptance test. They were copy/pasted in the `security_group_rule` acc tests.